packages/flutter_tools/test/web.shard/hot_reload_web_test.dart

```diff
@@ -4,6 +4,8 @@
 
 // @dart = 2.8
 
+@OnPlatform(<String, dynamic>{'windows': Timeout.factor(2)})
+
```

Comment:
nit: extra space here and below

Reply:
Would you prefer it removed or added more? I could not find any examples in flutter code.

Reply:
removed:)